### ISSUE TITLE
Customizable badge text

### ISSUE THREAD

**Issue by lbergelson** [jenkinsci/embeddable-build-status-plugin#22]
We have several different builds running for a given project: tests on every commit, slower tests that run nightly, tests to see if we're breaking related downstream repositories, etc.  Would it be possible to make the status icon text customizable so several badges could live side by side without confusion?

**Comment by homeles** [comment]
I have the same issue +1

**Comment by arpit-nagar** [comment]
:+1:

**Comment by DarthGandalf** [comment]
As a workaround, http://shields.io/ supports querying Jenkins via JSON and applying a custom label.
Of course, that wouldn't work for private installations. Also, it doesn't depend on this plugin.

**Comment by lbergelson** [comment]
@DarthGandalf Thanks for the suggestion.  I didn't know about that before.

**Comment by CelesteGH** [comment]
That would be nice.

**Comment by DeChrish** [comment]
Nice ++

**Comment by LJBD** [comment]
Is there a plan to do this? I'd also like to see that happening.

**Comment by actionless** [comment]
i think it would be nice to be able to customize the text just as a GET parameter, ie:

```
my-jenkins.com/buildStatus/icon?job=PROJ/application/branch&text=mytext
```

**Comment by thomas-dee** [comment]
Resolved with PR #31
